docker-compose.yml

```diff
@@ -72,8 +72,6 @@ services:
       lagoon.type: cli-persistent
       lagoon.persistent.name: nginx # mount the persistent storage of nginx into this container
       lagoon.persistent: /app/web/sites/default/files/ # location where the persistent storage should be mounted
-    environment:
-      << : *default-environment # loads the defined environment variables from the top
```

Comment:
Why this  change to docker setup?

Reply:
Lines were duplicated. See commmit https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/186/commits/c1f62c1e0f62620309deedaf33d6793b58d7cff1